src/foam/nanos/crunch/CapabilityCapabilityJunctionRefine.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'CapabilityCapabilityJunctionRefine',
+  refines: 'foam.nanos.crunch.CapabilityCapabilityJunction',
+
+  documentation: `
+    Refine capabilitycapabilityjunction to add tablecellformatters for source and target id
+  `,
+
+  properties: [
+    {
+      class: 'Reference',
+      of: 'foam.nanos.crunch.Capability',
+      name: 'sourceId',
+      tableCellFormatter: function(value, obj, axiom) {
```

Comment:
Just curious, did you try to add these tableCellFormatters to target and source property of the relationship?

Reply:
ya I tried that at first but i think those are actually for defining property properties of the property on the source/target models, instead of being related to the source/target properties of the junction
so an example of when you'd use source/targetproperty is : user's daosummaryview should show their capabilities but capability's daosummaryview should not show its users. so we set something like `sourceProperty:{visibility:'RO'}` to make the capabilities show up in the user model's daosummaryview